library/src/main/java/com/getstream/sdk/chat/adapter/MediaAttachmentSelectedAdapter.java

```diff
@@ -75,6 +75,13 @@ public void bind(Attachment attachment, final OnItemClickListener listener) {
                     .solidColor(Color.BLACK)
                     .cornerRadii(cornerRadius, cornerRadius, cornerRadius, cornerRadius)
                     .build());
+
+            binding.ivMask.setShape(context, new DrawableBuilder()
+                    .rectangle()
```

Comment:
File picker break if you pick one file and then try to pick another one. See screenshot for final state (I picked 1 image and then tapped on the same again)

![image](https://user-images.githubusercontent.com/88735/66919876-7d1b3480-f022-11e9-86fb-0d43275531bf.png)

Reply:
Fixed it